src/components/ListItem.jsx

```diff
@@ -4,9 +4,21 @@ export function ListItem({
 	name,
 	isDefaultChecked,
 	itemId,
+	urgency,
 	setIsChecked,
 	setCheckedItemId,
 }) {
+	const urgencyIcon =
+		urgency === 'overdue'
+			? '🔴'
+			: urgency === 'soon'
+			? '🟠'
+			: urgency === 'kind of soon'
+			? '🟡'
+			: urgency === 'not soon'
+			? '🟢'
+			: '⚫️';
```

Comment:
Something to think about as the team moves into stying work over the next couple of weeks - this icon is a bit difficult to see for browser's using dark mode. Color contrast is something to consider moving forward.

<img width="289" alt="Screenshot 2023-05-13 at 10 08 44 AM" src="https://github.com/the-collab-lab/tcl-57-smart-shopping-list/assets/78327387/ed047118-469d-46ca-babf-d781f75b7868">

Reply:
Yes! Also, because these icons are decorative, we probably want to hide them from assistive technology. A screen reader user might, for example, hear "Apples (orange circle soon)", but "orange circle" is extra noise in that case because it's a visual-only enhancement of "soon".